package.json

```diff
@@ -20,12 +20,12 @@
     "clean": "node ./scripts/compile-clean.js",
     "compile-icons": "node ./scripts/compile-icons.js && prettier --write --loglevel=warn \"./src/components/icon/assets/**/*.js\"",
     "extract-i18n-strings": "node ./scripts/babel/fetch-i18n-strings",
-    "lint": "yarn tsc --noEmit && yarn lint-es && yarn lint-sass",
+    "lint": "yarn tsc --noEmit && yarn lint-es && yarn lint-style",
     "lint-fix": "yarn lint-es-fix",
     "lint-es": "eslint --cache \"{src,src-docs}/**/*.{ts,tsx,js}\" --max-warnings 0",
     "lint-es-fix": "yarn lint-es --fix",
-    "lint-sass": "sass-lint -v --max-warnings 0",
-    "lint-sass-fix": "sass-lint-auto-fix -c ./.sass-lint-fix.yml",
+    "lint-style": "stylelint \"{src,src-docs}/**/*.scss\" --max-warnings 0",
+    "lint-style-fix": "yarn lint-style --fix",
```

Comment:
Curious if this will cause anything unintended to break. All CI passes, but I wonder if anyone else is using these commands that we don't know about. Either way, I don't think it's something we _really_ need to worry about for breaking things ~ this is more meant for dev anyway